index.js

```diff
@@ -32,7 +33,13 @@ console.log('hello world')
  * Example Output: "Elizabeth Thompson"
  */
 
-// TODO: ADD CODE HERE
+const firstName = 'Alex';
+const lastName = 'Rodriguez';
+console.log(firstName + ' ' + lastName);
```

Comment:
You could also use a template literal here!

Read more about [Template Literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals)

Reply:
Hi Elizabeth, Thank you so much for reviewing my code and for the comments, I just fixed them, can you please review it again if you don't mind. do i need to create a new pull request?